### DISCUSSION TITLE
Discord community call: 17 February 2024

### DISCUSSION THREAD

**Discussion by BenJamesBen** [freesewing/freesewing#5719] [Support] [answered]
You are all invited to our Discord hangout.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About

FreeSewing's Discord hangout is an informal hangout/chat/meeting that is open to everyone.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

### Details

Location: https://discord.freesewing.org/ (meeting room: 🔊voice chat)
Date: February 17th, 2024 5pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/feb-17-2024/5pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda

- TBD

**Comment by tangerineshark** [comment]
heads up that I won't be at this call! I'm taking a croissant making class! 🥐 🧡

**Comment by BenJamesBen** [comment, marked as answer]
February 17, 2024 notes:

- Are we getting a pattern for Toto the canvas tote bag? Maybe, but not in the near future.
- Things people are making and working on.
- A project to make 1 of every FreeSewing design?
- Extra-wide fabric to make Gozer? Bedsheets, quilt backing fabric.
- How was the wave shirt collar made? Lots of interfacing. How was the fabric printed? Spoonflower, awhile ago.
- A tour of RockerKitten's sewing room and sewing machines.
- What materials to buy in bulk? Muslin.
- Thoughts on Feature Request #6005 to add NSFW/spoiler functionality to website to hide images? General consensus that it's a good idea and that we should also provide guidelines about exactly which situations to apply it.
- Playing with clo3d and blender.

Attendees:
- Thrunic
- Benjamin F.
- Vili
- Wouter.vdub
- nataliasayan
- RockerKitten
- Darigov Research
- siart

Discord meeting chat contains photos and links about things discussed:
- https://discord.com/channels/698854858052075530/788066844393078835/1208442764304580690
- discord://discord.com/channels/698854858052075530/788066844393078835/1208442764304580690
